### DISCUSSION TITLE
How to configure Blue Iris

### DISCUSSION THREAD

**Discussion by xnorpx** [xnorpx/blue-onyx#46] [Documentation]
General Blue Iris setup:

https://youtu.be/jqc1ukrkMmI?si=DU0p3ZDRByN_z0gc&t=570

This is my settings:

Main settings:

![image](https://github.com/user-attachments/assets/a822e8df-9f55-4f5b-a262-22f30f3a77bd)

AI settings under alerts under camera settings

![image](https://github.com/user-attachments/assets/b0ebb8dc-e132-49ef-bb02-9580b9ff1693)

**Comment by ch3st3rk** [comment]
Quick question, if you were to use the ipcam-combined model for example and run this as a service, do you need to specify that in BI Custom Models, or is this still left empty?
I am thinking of trying ipcam-general during the day as this seems very fast but at night it misses cars vehicles that should be picked up quite easily.
So I am looking at running ipcam-general as a service in the day and the rt small model at night again as a service.
Is that possible and is it literally just a change to the service for the model or does BI need additional changes for each model?

Thanks

**Comment by xnorpx** [comment]
Yes Custom models should be empty.

So you probably need some external script and a time / task running in windows.

So I would do the following.

Install 2 services:

First service is called: BlueOnyxNight and you use ModelX
Second service is called BlueOnyxDay and you use the ModelY

have a script or 2 tasks that execute in evening and morning and switch the 2 services.

NightTine
StopService
StartOtherService  

Morning
StopOtherService
StartService

**Comment by ch3st3rk** [comment]
ok thanks would the port need changing if I created 2xservices? or can both be configured with the same port and then create a task to start one between time A&B and the other to start between time C&D?

I am wondering if I can create a cmd line like the below somehow and the repeat for each different model

sc.exe create blue_onyx_service_day binPath= "C:\.blue-onyx\blue_onyx_service.exe --port 38666" start= TIME Stop=TIME displayname= "Blue Onyx Service Day"

thanks

**Comment by xnorpx** [comment]
The 2 services can have the same port, but the time when they are started cannot overlap as only one process can access a specific port at a time.

**Comment by xnorpx** [comment]
Blue Onyx does not know how to stop itself or start it self based on tine. You need to figure out how to control that externally.

**Reply by ch3st3rk** [reply]
No problem thanks I have sorted that with Task Scheduler so will give it all a go thanks for your help!

**Comment by mythosmc** [comment]
Can you repost your setting please under the new version of blueiris?

Thank you,
